bsp4j/src/main/java/ch/epfl/scala/bsp4j/ScalaBuildServer.java

```diff
@@ -14,4 +14,7 @@
 
     @JsonRequest("buildTarget/scalaMainClasses")
     CompletableFuture<ScalaMainClassesResult> buildTargetScalaMainClasses(ScalaMainClassesParams params);
+
+    @JsonRequest("buildTarget/jvmTestEnvironment")
```

Comment:
Should be in a JVMBuildServer interface (which Scala one can possibly extend)

Reply:
Done